src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -550,6 +614,19 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
         return UTIL_Namespace.getNamespace() != '';
     }
 
+    /*******************************************************************************************************
+    * @description checks for read, create, and edit FLS for a given field
+    * @param dfr DescribeFieldResult of the field to check
+    * @return Boolean
+    */
+    private static Boolean checkFLS(DescribeFieldResult dfr) {
```

Comment:
Maybe a more descriptive name here, like canUpdateField()?

Reply:
This feels like a solid candidate to move to a FLS Util class as well, if one were to exist